### ISSUE TITLE
Unable install in pnpm

### ISSUE THREAD

**Issue by nodegin** [kamilkisiela/graphql-config#1366]
### Issue workflow progress

_Progress of the issue based on the [Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_

- [ ] 1. The issue provides a reproduction available on GitHub, Stackblitz or CodeSandbox
     > Make sure to fork this template and run `pnpm run generate` in the terminal.
     >
     > Please make sure the Codegen and plugins version under `package.json` matches yours.
- [ ] 2. A failing test has been provided
- [ ] 3. A local solution has been provided
- [ ] 4. A pull request is pending review

---

**Describe the bug**

**To Reproduce**

With latest @graphql-codegen/cli release, which uses graphql-config@5.0.1, I cannot install dependencies with `pnpm install` command,

<img width="837" alt="Screen Shot 2023-05-24 at 17 42 53" src="https://github.com/kamilkisiela/graphql-config/assets/8536244/42684125-613e-46e3-aace-f38345b437c7">

It seems this package has a mistake on package.json file where the `engines` specifies

```
  "engines": {
    "node": ">= 16.0.0",
    "pnpm": "7.32.4"
  }
```

forcing pnpm to be only able to install with v7.32.4. 

I guess the line is missing a `>=` operator?

**Expected behavior**

It shouldn't require explicitly v7.32.4 to install.

**Environment:**

- OS: macOS
- GraphQL Config Version: 5.0.1
- NodeJS: 20

**Additional context**

**Comment by nodegin** [comment]
PS: This issue probably only happen when you have a `.npmrc` file in your project with the following content:
```engine-strict=true```

**Comment by niklasnatter** [comment]
Thanks for reporting! We are running into this issue too and I think the only workaround is to remove the `engine-strict=true` for now.

**Comment by jstans** [comment]
Looks like this was caused by https://github.com/kamilkisiela/graphql-config/pull/1356 (switching from yarn to pnpm internally).

I'm not sure if the engine flag should be needed though, especially since it was never used for any other package manager. Perhaps a PR to make it `">=7.32.4"` at least is in order; if not removing it entirely?

Personally I make use of the engine-strict flag to prevent accidental installs via npm, so removing `engine-strict` isn't ideal.

**Comment by dotansimha** [comment]
We are working on a fix. We'll ship a new version without this constraint soon.

**Comment by dotansimha** [comment]
Fixed in `graphql-config@5.0.2`. We'll do an upstream fix for codegen and other packages that depends on this one.
